packages/web-components/fast-foundation/src/radio-group/radio-group.template.ts

```diff
@@ -12,15 +12,16 @@ export const radioGroupTemplate: FoundationElementTemplate<ViewTemplate<RadioGro
     context,
     definition
 ) => html`
-    <template
+    <div
```

Comment:
I'm not sure I'm ready to have this be the resolution or that it's doable as a backwards compat fix. This breaks the template and would be a breaking change.